src/pages/guides/validators.md

```diff
@@ -3,11 +3,94 @@ title: Validators and Staking
 metaTitle: Validators and Staking | Guides
 description: An overview of Solana Validators and staking mechanics.
 # remember to update dates also in /components/guides/index.js
-created: '06-16-2024'
-updated: '06-21-2024'
+created: '04-19-2025'
+updated: '04-19-2025'
 ---
 ## Overview
 
 Validators are responsible for processing transactions, generating new blocks, and validating the state of the blockchain to ensure accuracy and prevent double-spending. They participate in the consensus mechanism by voting on the legitimacy of blocks proposed by other validators, which helps to maintain the integrity and security of the network. Validators also contribute to the network's decentralization by staking their SOL tokens, which aligns their incentives with the network's health and stability.
 
-Validators often participate in governance decisions, providing insights and voting on proposals that affect the network's future. Many validators also contribute to the community by offering educational resources, running community nodes, and supporting the development of decentralized applications (dApps) and tools that enhance the ecosystem.
\ No newline at end of file
+Validators often participate in governance decisions, providing insights and voting on proposals that affect the network's future. Many validators also contribute to the community by offering educational resources, running community nodes, and supporting the development of decentralized applications (dApps) and tools that enhance the ecosystem.
+
+## Solana's Validator Network
+
+### Proof of Stake and Proof of History
+
+Solana uses a unique combination of Proof of Stake (PoS) and Proof of History (PoH) consensus mechanisms:
+
+- **Proof of Stake**: Validators must stake SOL tokens to participate in consensus. The amount staked influences their voting weight and potential rewards.
+- **Proof of History**: A cryptographic clock that provides a historical record of events, allowing validators to agree on the timing of events without requiring communication.
+
+This hybrid approach enables Solana to achieve high transaction throughput (up to 65,000 TPS) and low latency (400ms block times).
+
+### Validator Requirements
+
+Running a validator on Solana requires:
+
+- Hardware: High-performance servers with substantial computational power, including:
+  - CPU: 12 cores / 24 threads, 2.8GHz or faster
+  - RAM: 256GB or more
+  - Storage: 2TB or larger NVMe SSD
+  - Network: 1 Gbps or faster internet connection
+- Software: Solana validator software and supporting tools
+- SOL Tokens: To cover vote transaction fees and optional self-stake
+
+### Validator Economics
+
+Validators earn rewards through:
+
+1. **Transaction Fees**: A portion of transaction fees paid by users
+2. **Inflation Rewards**: New SOL tokens distributed to validators and delegators
+3. **MEV (Maximal Extractable Value)**: Additional value that can be extracted by influencing transaction ordering
```

Comment:
Not sure if we should add stuff like that. While it might be the case now it also might change in the future which would require us to update it